### PR TITLE
Implement missing feeding subscriptions for moderators

### PR DESCRIPTION
**Pull request by Fileek**
### Description
- Implemented missing feeding subscriptions for moderators  
- Added `createdAt` filter for `getAssignedFeedings()`

### CODE REVIEW

File: library/api/src/main/java/com/epmedu/animeal/api/feeding/FeedingApi.kt [hunk 1]
```diff
@@ -26,6 +27,7 @@ interface FeedingApi {
         feedingPointId: String? = null,
         assignedModeratorId: String? = null,
         status: FeedingStatus? = null,
+        createdAt: SearchableStringFilterInput? = null
```
Comment (mskibin) [inline library/api/src/main/java/com/epmedu/animeal/api/feeding/FeedingApi.kt:31]: Please add it to docs

File: library/api/src/main/java/com/epmedu/animeal/api/feeding/FeedingHistoryApi.kt [hunk 2]
```diff
@@ -24,7 +25,8 @@ interface FeedingHistoryApi {
     suspend fun getFeedingHistoriesBy(
         feedingPointId: String? = null,
         assignedModeratorId: String? = null,
-        status: FeedingStatus? = null
+        status: FeedingStatus? = null,
+        createdAt: SearchableStringFilterInput? = null
```
Comment (mskibin) [inline library/api/src/main/java/com/epmedu/animeal/api/feeding/FeedingHistoryApi.kt:30]: Also should add to docs

**Review by mskibin — APPROVED** [review]

File: library/api/src/main/java/com/epmedu/animeal/api/feeding/FeedingApi.kt (hunk 1 above)
Comment (Fileek) [reply library/api/src/main/java/com/epmedu/animeal/api/feeding/FeedingApi.kt:31]: Good catch, done.

File: library/api/src/main/java/com/epmedu/animeal/api/feeding/FeedingHistoryApi.kt (hunk 2 above)
Comment (Fileek) [reply library/api/src/main/java/com/epmedu/animeal/api/feeding/FeedingHistoryApi.kt:30]: Also done :)

**Pull request merged**